vc_zoom/indico_vc_zoom/plugin.py

```diff
@@ -501,6 +462,12 @@ def _merge_users(self, target, source, **kwargs):
         ):
             room.data['host'] = target.identifier
             flag_modified(room, 'data')
+        for room in VCRoom.query.filter(
+            VCRoom.type == self.service_name, VCRoom.data.contains({'alternative_hosts': [source.identifier]})
```

Comment:
Will this contains filterwork when there are already multiple `alternative_hosts`, ie is it a deep contains or does it only check the top-level json object for containment, and then for an exact value when it comes to the list in there?

Reply:
I did test the query and it finds rooms also with more than one alternative host.